app-devel/intel-oneapi-basekit/autobuild/defines

```diff
@@ -6,9 +6,3 @@ PKGDES="Intel oneAPI base toolkit"
 FAIL_ARCH="!(amd64)"
 ABSTRIP=0
 NOSTATIC=0
-
-# FIXME: currently an OpenMP GPU program compiled with "-g" flag will trigger a runtime error:
-# InvalidBuiltinSetName: Expects OpenCL12, OpenCL20. Actual is NonSemantic.Shader.DebugInfo.200 [Src: /var/cache/acbs/build/acbs.kjwza1v_/intel-graphics-compiler/IGC/AdaptorOCL/SPIRV/libSPIRV/SPIRVModule.cpp:565 SPIRVBuiltinSetNameMap::rfind(BuiltinSetName, &BuiltinSet) ]
-# terminate called after throwing an instance of 'std::runtime_error'
-#   what():  internal compiler error
-# Aborted (core dumped)
```

Comment:
Is the problem solved? I didn't see any change to the build file.

Reply:
Yes, it's been solved. The root cause is the dependency IGC, which was fixed in PR #8906.